projects/1.16/assets/pneumaticcraft-repressurized/pneumaticcraft/lang/zh_cn.json

```
@@ -1466,35 +1466,35 @@
-    "pneumaticcraft.gui.tab.problems.gasLift.stuck": "§f钻管被阻挡！${br}§0钻管遇到了无法被破坏的方块，无法继续向下挖掘。",
-    "pneumaticcraft.gui.tab.problems.gasLift.noTubeSpace": "§f没有空间可供钻管！${br}§0从气举中移除多余的钻管。",
-    "pneumaticcraft.gui.tab.problems.gasLift.noTubes": "§f没有钻管！${br}§0向气举中添加钻管。",
+    "pneumaticcraft.gui.tab.problems.gasLift.stuck": "§f钻管被阻挡${br}§0钻管遇到了无法被破坏的方块，无法继续向下挖掘。",
+    "pneumaticcraft.gui.tab.problems.gasLift.noTubeSpace": "§f没有空间可供钻管${br}§0从气举中移除多余的钻管。",
+    "pneumaticcraft.gui.tab.problems.gasLift.noTubes": "§f没有钻管${br}§0向气举中添加钻管。",
     "pneumaticcraft.gui.tab.problems.etching_tank.no_acid": "§f没有蚀刻酸${br}§0向蚀刻器添加蚀刻酸。",
     "pneumaticcraft.gui.tab.problems.etching_tank.failed_full": "§f失败输出满${br}§0在失败物品输出被清空前，没有物品可以被蚀刻。",
     "pneumaticcraft.gui.tab.problems.elevator.fully_extended": "§f完全升高${br}§0向电梯上方放置（更多）电梯框架。",
     "pneumaticcraft.gui.tab.problems.electrostatic.notEnoughGrounding": "§f空气重定向${br}§0若压缩机在内部空气满时被闪电击中，并非所有能量都可以被重定向！向静电压缩机的下方连接更多铁栅栏。${br}接地：%d/%d mL",
     "pneumaticcraft.gui.tab.problems.charging_station.pressure_equal": "§f%s无法被充/放气${br}§0物品与充气站的气压已经平衡。",
-    "pneumaticcraft.gui.tab.problems.charging_station.no_item": "§f没有可充/放其的物品${br}§0向充气槽位放置气动物品。",
-    "pneumaticcraft.gui.tab.problems.assembly_controller.no_program": "§f没有程序！${br}§0向装配控制器放入一个装配程序。",
-    "pneumaticcraft.gui.tab.problems.assembly_controller.missingMachine": "§f缺失机器！${br}§0并非所选程序所需的每一台机器都存在。向装配线添加一台%s。",
-    "pneumaticcraft.gui.tab.problems.assembly_controller.duplicateMachine": "§f检测到重复机器！${br}§0确保只有一台%s。",
-    "pneumaticcraft.gui.tab.problems.amadron.tooMuchFluid": "§f订购流体量过大！${br}§0一台亚马龙无人机的最大装载流体量为 36 桶。你需要把订单分为多个订单。",
-    "pneumaticcraft.gui.tab.problems.amadron.notEnoughStock": "§f库存不足！${br}§0这已经是最大供货量！",
-    "pneumaticcraft.gui.tab.problems.amadron.tooManyItems": "§f订购物品量过大！${br}§0一台亚马龙无人机的最大装载物品量为 36 组。你需要把订单分为多个订单。",
-    "pneumaticcraft.gui.tab.problems.amadron.outOfStock": "§f库存空！${br}§0稍等，或许供货商会补足库存...",
-    "pneumaticcraft.gui.tab.problems.amadron.notEnoughItemSpace": "§f目标容器没有足够的空间。${br}§在绑定的物品容器中清出足够的空间。",
-    "pneumaticcraft.gui.tab.problems.amadron.notEnoughFluidSpace": "§f目标容器没有足够的空间。${br}§0确保绑定的容器中没有其余流体，或者绑定更大的流体容器。",
-    "pneumaticcraft.gui.tab.problems.amadron.notEnoughItems": "§f没有可售出的物品！${br}§0订单量已到最大。若需要，向绑定的容器加入更多物品。",
-    "pneumaticcraft.gui.tab.problems.amadron.notEnoughFluid": "§f没有可售出的流体！${br}§0订单量已到最大。若需要，向绑定的容器加入更多流体。",
-    "pneumaticcraft.gui.tab.problems.amadron.noItemProvider": "§f未选择物品存储。${br}§0使用亚马龙平板电脑右击物品存储。",
-    "pneumaticcraft.gui.tab.problems.amadron.noFluidProvider": "§f未选择流体存储。${br}§0使用亚马龙平板电脑右击流体存储。",
-    "pneumaticcraft.gui.tab.problems.airLeak": "§f空气泄漏！${br}§0在输出面连接管道或机器。",
+    "pneumaticcraft.gui.tab.problems.charging_station.no_item": "§f没有可充/放气的物品${br}§0向充气槽位放置气动物品。",
+    "pneumaticcraft.gui.tab.problems.assembly_controller.no_program": "§f没有程序${br}§0向装配控制器放入一个装配程序。",
+    "pneumaticcraft.gui.tab.problems.assembly_controller.missingMachine": "§f缺失机器${br}§0并非所选程序所需的每一台机器都存在。向装配线添加一台%s。",
```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
    "pneumaticcraft.gui.tab.problems.assembly_controller.missingMachine": "§f缺失机器${br}§0缺失部分所选程序所需的机器。请向装配线添加一台%s。",
```